lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -11,11 +11,13 @@ def self.run(params)
         # Checkout develop and update
         Fastlane::Helper::GitHelper.checkout_and_pull('develop')
 
+        @app = ENV['APP'].nil? ? params[:app] : ENV['APP']
```

Comment:
`self.run` is a static method not an instance method, so using `@app` aka an instance variable in that context seems incorrect to me 🤔  (wonder if rubocop will warn about this)